pkg/webui/console/components/duration-input/index.js

```diff
@@ -0,0 +1,118 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import classnames from 'classnames'
+import bind from 'autobind-decorator'
+
+import Select from '../../../components/select'
+import Input from '../../../components/input'
+import { unit as unitRegexp } from '../../lib/regexp'
+import PropTypes from '../../../lib/prop-types'
+import style from './duration-input.styl'
+
+class DurationInput extends React.PureComponent {
```

Comment:
```suggestion
class UnitInput extends React.PureComponent {
```
It doesn't have to be used for durations only.